src/navigator.js

```diff
@@ -276,8 +276,8 @@ $.extend( $.Navigator.prototype, $.EventSource.prototype, $.Viewer.prototype, /*
     /**
       /* Flip navigator element
       */
-    toggleFlip: function() {
-      this.viewport.flipped = !this.viewport.flipped;
+    setFlip: function() {
+      this.viewport.toggleFlip();
```

Comment:
This should be `this.viewport.setFlip(state);` (and of course the state should be passed in to this function).

Reply:
Done